client/api/events.ts

```diff
@@ -70,9 +71,9 @@ export function searchEventsGetAll(params: ISearchParams): Promise<Array<IEventI
     });
 }
 
-export function getEventById(eventId: IEventItem['_id']): Promise<IEventItem> {
+export function getEventById(eventId: IEventItem['_id'], params: IGetRequestParams): Promise<IEventItem> {
     return superdeskApi.dataApi
-        .findOne<IEventItem>('events', eventId)
+        .findOne<IEventItem>('events', eventId + (params?.cache === false ? `?time=${Math.floor(Date.now() / 1000)}` : ''))
```

Comment:
Aren't we using the `Cache-Control: no-cache` header in our responses? Meaning if every hop along the network right through to the browser should abide by this response header value.

Is this not the case?

Reply:
we do, but that doesn't [disable caching actually](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control#no-cache), it makes a conditional request, and then it gets 304 from server due to last updated time / etag not changed

Reply:
Ah, so it's Eve that's doing that. Okay